dynamic_rest/serializers.py

```diff
@@ -504,11 +568,44 @@ def _faster_to_representation(self, instance):
         ret = {}
         fields = self._readable_fields
 
+        is_fast = isinstance(instance, prefetch.FastObject)
+        id_fields = self._readable_id_fields
+
         for field in fields:
-            try:
-                attribute = field.get_attribute(instance)
-            except SkipField:
-                continue
+            # we exclude dynamic fields here because the proper fastquery
+            # dereferencing happens in the `get_attribute` method now
+            if (
+                is_fast and
+                not isinstance(
```

Comment:
This check would happen for each field on each object instance being serialized. Probably not a huge overhead, but would be more optimal to pre-select the non-(DGRF,DRF) fields once at the serializer level, then loop over those fields.

Reply:
I'll do this one as an optimization in a follow-up